react-list.es6

```diff
@@ -224,14 +224,31 @@ module.exports = class ReactList extends Component {
     this.frameRequested = window.requestAnimationFrame(this.updateFrame);
   }
 
+  waitForFrameUpdate() {
```

Comment:
Instead of doing this via a promise -- would it be easier / cleaner to take a `onFrameUpdate` callback in props?
Then you could cut having the need to keep track of the promise.